dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/context/ContextManager.java

```diff
@@ -1056,6 +1063,17 @@ public InitializeResult initialize() throws WIMApplicationException {
             iEnvironment.put(LDAP_ENV_PROP_READ_TIMEOUT, String.valueOf(DEFAULT_READ_TIMEOUT));
         }
 
+        /*
+         * Enabled JNDI BER output if required.
+         */
+        if (iJndiOutputEnabled != null && iJndiOutputEnabled) {
+            synchronized (iLock) {
+                BEROutputStream berOS = new BEROutputStream();
+                PrintStream Stdout = new PrintStream(berOS);
```

Comment:
Why do we need to wrap with a PrintStream? BEROutputStream is already an java.io.OutputStream.